lib/krane/cli/krane.rb

```diff
@@ -31,6 +32,14 @@ def restart(namespace, context)
         end
       end
 
+      desc("run NAMESPACE CONTEXT", "Run a pod that exits upon completing a task")
+      expand_options(RunCommand::OPTIONS)
+      def run_command(namespace, context)
```

Comment:
Looks great to me! Just a nit, I feel it should be call `run` instead of `run_command`.

Reply:
We can't, `run` is a protected method name within Thor so this needs to be called something like `run_task` or `run_command` 😞 @jessie-JNing